modules/aws_asg/cloud-init.bash

```diff
@@ -21,10 +21,10 @@ main() {
     if [[ -f "${RUNDIR}/instance-ipv4" ]]; then
       local ipv4_label
       ipv4_label="travis.ipv4:$(cat "${RUNDIR}/instance-ipv4")"
-      container_labels="${container_labels} ${ipv4_label}"
+      container_labels="${container_labels}\\ ${ipv4_label}"
```

Comment:
I'm referring to this double backslash

Reply:
@Lyoness I tried with only one backslash, but that seems to result in the same behavior as pasted into the PR description unfortunately

Reply:
Maybe this would be easier if we switched to the ","-delimited ":"-paired values like I'd thought it was supposed to be :sweat_smile:   I realllllly don't like multiple levels of escaping and I'd much rather avoid it if possible.

Reply:
@meatballhat I totally agree. Can we still merge this for now so that container labels are parsed correctly until worker is updated to look for comma-delimited labels?

Reply:
Update: I've opened [a PR](https://github.com/travis-ci/worker/pull/445) to address this in worker.